pubspec.lock

```diff
@@ -21,35 +21,28 @@ packages:
       name: characters
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.2.0"
-  charcode:
-    dependency: transitive
-    description:
-      name: charcode
-      url: "https://pub.dartlang.org"
-    source: hosted
-    version: "1.3.1"
+    version: "1.2.1"
```

Comment:
let's not change pubspec.lock

Reply:
I am not manually changing it. I think it's got to do with my defualt flutter settings